.github/workflows/release.yml

```diff
@@ -23,16 +23,14 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install -U pip
-          python -m pip install -U setuptools twine wheel
+          python -m pip install -U poetry
+
       - name: Build package
         run: |
-          python setup.py --version
-          python setup.py sdist --format=gztar bdist_wheel
-          twine check dist/*
+          poetry build
+
       - name: Upload packages to Jazzband
         if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@master
-        with:
-          user: jazzband
-          password: ${{ secrets.JAZZBAND_RELEASE_KEY }}
-          repository_url: https://jazzband.co/projects/django-invitations/upload
+        run: |
+          poetry config repositories.jazzband https://jazzband.co/projects/django-invitations/upload
+          poetry config http-basic.jazzband jazzband ${{ secrets.JAZZBAND_RELEASE_KEY }}
```

Comment:
In fact, this won't actually upload the file, @valberg! It just configures it.

Reply:
@valberg, we're missing a publish step:

```yml
    - name: Upload packages to Jazzband
        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
        uses: pypa/gh-action-pypi-publish@master
        with:
          user: jazzband
          password: ${{ secrets.JAZZBAND_RELEASE_KEY }}
          repository_url: https://jazzband.co/projects/wagtailmenus/upload 
```

(I'm curious to see if building it with poetry will work. Perhaps using `poetry publish` instead could work too?)

Reply:
https://github.com/jazzband/django-invitations/pull/195 fixed this FWIW